webpack.config.babel.js

```diff
@@ -60,20 +68,29 @@ const baseStyleLoaders = [
   }
 ];
 
-const ONFIDO_API_URL = ENV === 'production' ? 'https://api.onfido.com' : 'https://apidev.onfido.com'
-const ONFIDO_SDK_SERVER_URL = ENV === 'production' ? 'https://sdk.onfido.com' : 'https://sdk-staging.onfido.com'
+const PROD_CONFIG = {
+  'ONFIDO_API_URL': 'https://api.onfido.com',
+  'ONFIDO_SDK_URL': 'https://sdk.onfido.com',
+  'JWT_FACTORY': 'https://sdk-jwt-factory-production.herokuapp.com/api/v2'
+}
+
+const STAGING_CONFIG = {
+  'ONFIDO_API_URL': 'https://apidev.onfido.com',
+  'ONFIDO_SDK_URL': 'https://sdk-staging.onfido.com',
+  'JWT_FACTORY': 'https://sdk-jwt-factory-staging.herokuapp.com/api/v2'
+}
+
+const CONFIG = PRODUCTION_API ? PROD_CONFIG : STAGING_CONFIG
 
 const basePlugins = [
   new webpack.NoEmitOnErrorsPlugin(),
   new webpack.DefinePlugin({
-    'process.env': {
-      'NODE_ENV': JSON.stringify(ENV),
-      'ONFIDO_API_URL': JSON.stringify(ONFIDO_API_URL),
-      'ONFIDO_SDK_SERVER_URL': JSON.stringify(ONFIDO_SDK_SERVER_URL),
-      'SDK_VERSION': JSON.stringify(packageJson.version)
-    }
+    'process.env.NODE_ENV': JSON.stringify(WEBPACK_ENV),
```

Comment:
@rfreitas Do you know why we have to set this? I could not work out which dependency relies on it.